frontend/src/app/components/canvas/canvas.component.ts

```diff
@@ -118,6 +119,7 @@ export class CanvasComponent implements OnInit, OnDestroy {
       [SocketEvent.BOARD_TAGS_UPDATE, this.handleBoardTagsUpdateEvent],
       [SocketEvent.BOARD_TASK_UPDATE, this.handleBoardTaskUpdateEvent],
       [SocketEvent.BOARD_UPVOTE_UPDATE, this.handleBoardUpvoteUpdateEvent],
+      [SocketEvent.VOTES_CLEAR, this.handleVotesClearEvent],
```

Comment:
I think you also need to add a listener for this inside `list-modal.component.ts` and `buckets-modal.component.ts`.